frontend/src/main/web/lib/stores/testData/glossary/entries.json

```diff
@@ -3,15 +3,13 @@
     {
       "id": 34,
       "pos": "",
-      "description": "1",
+      "description": "asdfhsbfsubfsiufhsiduhwicuhwiduhddddddddddddddddddddddddddddddddd sddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd dddddddddd",
```

Comment:
What is going on here? I guess just test data, but hard to tell if it is displaying properly (e.g. if it didn't show `dd` on the end).